hawkular-wildfly-monitor/src/main/java/org/hawkular/agent/monitor/service/MonitorService.java

```diff
@@ -512,11 +518,18 @@ private void startScheduler() throws Exception {
             try {
                 for (DMRInventoryManager im : this.dmrServerInventories.values()) {
                     resourceManager = im.getResourceManager();
+                    InventoryStorage invStorage = inventoryStorageProxy;
+                    try {
+                        invStorage = new ServerAddressResolver(im.getManagedServer(), inventoryStorageProxy);
+                    } catch (UnknownHostException e) {
+                        MsgLogger.LOG.error("Unable to get an IP address for managed server [" + im.getManagedServer()
+                                + "]", e);
```

Comment:
+1. Yes, `*f()` is also my personal preference. I must have blindly copied it from somewhere. And now when looking at `ServerAddressResolver` again, the constructor does not need to throw anything anymore. Let me fix it.